arch/arm/src/nrf53/nrf53_uid.h

```diff
@@ -0,0 +1,36 @@
+/****************************************************************************
+ * arch/arm/src/nrf53/nrf53_uid.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_ARM_SRC_NRF53_NRF53_UID_H
+#define __ARCH_ARM_SRC_NRF53_NRF53_UID_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdint.h>
+
+/****************************************************************************
+ * Public Function Prototypes
+ ****************************************************************************/
+
+void nrf53_get_uniqueid(uint8_t uniqueid[]);
```

Comment:
```suggestion
void nrf53_get_uniqueid(uint8_t uniqueid[8]);
```
Optional, just for readability.

Reply:
done, fixed also for nrf52